ibm-messaging-mq-cloud-showcase-app/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This a duplicate of the license at the root of the dev-patterns repository. Can be removed from here.